src/main/resources/AntwoordenModel_v2_2_2.csv

```diff
@@ -9,20 +9,20 @@ Ja,Ja,Nee,Ja,,,,,,,,,,,,,,3ae,N,Nee,Nee,Nee,Nee,301,gezag is niet te bepalen omd
 Ja,Ja,Nee,,,,,,,,,,,,,,,3ee,N,Nee,Nee,Nee,Nee,310,gezag is niet te bepalen omdat geboortegegevens (in onderzoek staan. 
 Ja,Ja,Ja,,,,,,,,,,,,,,,3,N,Nee,Nee,Nee,Nee,350,gezag is niet te bepalen omdat minderjarige buiten Nederland heeft verbleven.
 Ja,Ja,Nee,Ja,Nee,,,,,,,,,,,,,3a,N,Nee,Nee,Nee,Nee,351,gezag is niet te bepalen omdat minderjarige in het buitenland is geboren en geen Nederlandse adoptie-akte bekend is. 
-Ja,Ja,Nee,Nee,,,,,,,,,,,,,,4e,N,Nee,Nee,Nee,Nee,400,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx  
-Ja,Ja,Nee,Ja,Ja,,,,,,,,,,,,,4ae,N,Nee,Nee,Nee,Nee,401,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx  
-Ja,Ja,Nee,Nee,,Ja,,,,,,,,,,,,4ee,N,Nee,Nee,Nee,Nee,420,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx 
-Ja,Ja,Nee,Ja,Ja,Ja,,,,,,,,,,,,4aee,N,Nee,Nee,Nee,Nee,421,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx 
-Ja,Ja,Nee,Nee,,Ja,,,,,,Nee,,,,,,4eee,N,Nee,Nee,Nee,Nee,430,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx 
-Ja,Ja,Nee,Ja,Ja,Ja,,,,,,Nee,,,,,,4aeee,N,Nee,Nee,Nee,Nee,431,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: xxx xxx xxx xxx 
+Ja,Ja,Nee,Nee,,,,,,,,,,,,,,4e,N,Nee,Nee,Nee,Nee,400,gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: %s
```

Comment:
@CathyDingemanse  @Patrick-4488 Hiervoor zie ik nog een aantal situaties van gezag niet te bepalen waar denk ik nog iets te verbeteren valt:
* regel 4: uitleg begint met spatie
* regel 7: hier staat alleen "gezag is niet te bepalen, omdat???" zonder verdere toelichting. Ook zorgt de komma ervoor dat het geen correcte csv meer is (de komma is ook kolom/veld-scheiding).

Reply:
detail: soms eindigt de toelichting op een punt, soms niet.

Reply:
Kan uiteraard worden aangepast. Bij de omdat??? had ik geen enkel idee wat de oorzaak was. Kunnen wij die achterhalen?